endpoints/openrtb2/sample-requests/valid-whole/exemplary/all-ext.json

```diff
@@ -65,6 +65,10 @@
         "cache": {
           "bids": {}
         },
+        "channel" : {
```

Comment:
Please correct the indentation here

Reply:
Interesting, in my VS Code, the indentation is correct and not really spaced out as it is here. If you pull down this code, is the indentation also incorrect locally?